lib/matplotlib/transforms.py

```diff
@@ -1389,19 +1389,19 @@ def contains_branch(self, other):
                 return True
         return False
 
-    def contains_branch_seperately(self, other_transform):
+    def contains_branch_separately(self, other_transform):
```

Comment:
api break?

Reply:
Yes. Please do not modify any names that could technically be available publically (local variables would be ok). We have to carefully consider if we want to change these, and if we do, we have deprecation machinery we have to go through. Any API-changing suggestions should be handled in a separate issue/PR, because that likely needs a discussion, and that should not be mixed with the trivial mass-changes in this PR.